multimodal/src/autogluon/multimodal/models/utils.py

```diff
@@ -692,19 +692,26 @@ def run_model(model: nn.Module, batch: dict):
     from .timm_image import TimmAutoModelForImagePrediction
 
     supported_models = (TimmAutoModelForImagePrediction, HFAutoModelForTextPrediction)
-    if (not isinstance(model, DocumentTransformer)) and isinstance(model, supported_models):
-        input_vec = [batch[k] for k in model.input_keys]
+
+    is_dp = True if isinstance(model, nn.DataParallel) else False
+
+    pure_model = model.module if is_dp else model
```

Comment:
nitpick: `isinstance` should return the boolean value needed by `is_dp`, therefore, these can be merged into a one-liner.

Reply:
Good call. Updated.